packages/code-block/docs.mdx

```diff
@@ -137,19 +138,61 @@ hello world
-| Multi-line with multiple commands | ❌        | Not supported. Only the first command will be copied.             | 🛑                                                                                                                   |
+| Scenario                                   | Supported | What gets copied                                                                                                                                                                                   | Example                                                                                                              |
+| ------------------------------------------ | --------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------- |
+| Single line commands                       | ✅        | The command will be copied without the leading `$`.                                                                                                                                                | [🔗](https://learn.hashicorp.com/vault/secrets-management/sm-static-secrets#step-5-retrieve-the-secrets)             |
+| Multi-line commands                        | ✅        | The multi-line command will be copied, without the leading `$`. Do not include lines after the multi-line command, such as output or additional commands, as they will not be processed correctly. | [🔗](https://learn.hashicorp.com/vault/secrets-management/sm-static-secrets#q-how-do-i-save-multiple-values-at-once) |
+| Commands with output                       | ✅        | The command will be copied without the leading `$` or the output.                                                                                                                                  | [🔗](https://learn.hashicorp.com/vault/secrets-management/sm-static-secrets#additional-discussion)                   |
+| Non-shell Snippets                         | ✅        | The entire snippet will be copied.                                                                                                                                                                 | [🔗](https://learn.hashicorp.com/nomad/getting-started/jobs#modifying-a-job)                                         |
+| Single-line commands across multiple lines | ✅        | Commands will be copied without their leading `$ `. Lines that do not start in `$ ` will be omitted from the copied snippet.                                                                       | ⏳ (none yet, relatively new feature)                                                                                |
```

Comment:
Updated this table, we now support many single-line commands in a single `shell-session` snippet 👍